### PR TITLE
Update content from 2008

### PR DESCRIPTION
**Pull request by estelle**

### CODE REVIEW

File: files/en-us/web/api/svgaltglyphelement/format/index.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ browser-compat: api.SVGAltGlyphElement.format
 The **`SVGAltGlyphElement.format`** property is a
 string that defines the format of the given font. It has the same
 meaning as the 'format' property of {{domxref("SVGGlyphRefElement")}} property. If the
-font is in one of the formats listed in [CSS2(\[CSS2\],
-section15.3.5)](https://www.w3.org/TR/2008/REC-CSS2-20080411/fonts.html#referencing), then its value is the corresponding \<string> parameter
+font is in one of the formats listed in [CSS2
+section 15.3.5)](https://www.w3.org/TR/2008/REC-CSS2-20080411/fonts.html#referencing), then its value is the corresponding \<string> parameter
 of the font.
```
Comment (hamishwillee) [inline files/en-us/web/api/svgaltglyphelement/format/index.md:17]:
```suggestion
font is in one of the formats listed in [CSS2 section 15.3.5)](https://www.w3.org/TR/2008/REC-CSS2-20080411/fonts.html#referencing), then its value is the corresponding \<string> parameter of the font.
```

File: files/en-us/web/api/svgaltglyphelement/format/index.md [hunk 2]
```diff
@@ -12,9 +12,7 @@ browser-compat: api.SVGAltGlyphElement.format
 The **`SVGAltGlyphElement.format`** property is a
 string that defines the format of the given font. It has the same
 meaning as the 'format' property of {{domxref("SVGGlyphRefElement")}} property. If the
-font is in one of the formats listed in [CSS2(\[CSS2\],
-section15.3.5)](https://www.w3.org/TR/2008/REC-CSS2-20080411/fonts.html#referencing), then its value is the corresponding \<string> parameter
-of the font.
+font is in one of the formats listed in [CSS2 section 15.3.5)](https://www.w3.org/TR/2008/REC-CSS2-20080411/fonts.html#referencing), then its value is the corresponding \<string> parameter of the font.
```
Comment (hamishwillee) [inline files/en-us/web/api/svgaltglyphelement/format/index.md:15]:
```suggestion
font is in one of the formats listed in [CSS2 section 15.3.5](https://www.w3.org/TR/2008/REC-CSS2-20080411/fonts.html#referencing), then its value is the corresponding \<string> parameter of the font.
```

**Review by hamishwillee — APPROVED** [review]
Looks good to me. Not merging though because I don't know enough to understand why some of the information was removed.

**Comment by estelle** [comment]
> Looks good to me. Not merging though because I don't know enough to understand why some of the information was removed.

::selection was originally a spec nightmare, but has been supported now for a while. The note was kind of "this is why you might not see it in the spec", but that is no longer the case and hasn't been for 4 years or so.

**Comment by hamishwillee** [comment]
Thanks for the explanation!

**Pull request merged**